File: examples/example001/example001.cs
using System;
using Pigmeo;
using Pigmeo.Extensions;
using Pigmeo.MCU;

namespace Example001 {
	class Program {
		static byte foo;

		static void Main(string[] args) {
			foo = Registers.PORTA;
			while(true) {
				foo++;
			}
		}
	}
}

File: old-code/examples/example001/example001.vb
Imports Pigmeo
Imports Pigmeo.Extensions
Imports Pigmeo.MCU

Module Module1
	Dim foo As Byte

	Sub Main()
		foo = Registers.PORTA
		While True
			foo += 1 'Note: by default this operation checks overflows, so Pigmeo Compiler doesn't generate exactly the same assembly language code for example001.cs and example001.vb
		End While
	End Sub

End Module
